examples/all-clusters-app/nrfconnect/main/AppTask.cpp

```diff
@@ -36,13 +36,18 @@
 #include <credentials/DeviceAttestationCredsProvider.h>
 #include <credentials/examples/DeviceAttestationCredsExample.h>
 
+#ifdef CONFIG_CHIP_WIFI
+#include <app/clusters/network-commissioning/network-commissioning.h>
+#include <platform/nrfconnect/wifi/NrfWiFiDriver.h>
+#endif
+
 #if CONFIG_CHIP_OTA_REQUESTOR
 #include "OTAUtil.h"
 #endif
 
 #include <dk_buttons_and_leds.h>
-#include <zephyr/logging/log.h>
```

Comment:
Why did you remove `zephyr/` in many places?

Reply:
To support sdk-nrf-next.

Reply:
Ok, I see. I suspected so, but wanted to make sure. We should remember to put it back in case of merging it to the public master branches.

Reply:
Sure, I think that when we change the revision to proper sdk-nrf commit this branch will simply not build without reverting these changes.